src/weibull.js

```diff
@@ -0,0 +1,15 @@
+import defaultSource from "./defaultSource.js";
+
+export default (function sourcerandomWeibull(source) {
+  function randomWeibull(lambda, k) {
+    if ((k = +k) < 0) throw new RangeError("invalid k");
+    k = 1 / k;
+    return function() {
+      return lambda * Math.pow(-Math.log1p(-source()), k);
+    };
+  }
+
+  randomWeibull.source = sourcerandomWeibull;
```

Comment:
```suggestion
  randomWeibull.source = sourceRandomWeibull;
```

Reply:
Corrected, and then some. The basic functionality of `randomWeibull` has been preserved.